scripts/build.js

```diff
@@ -133,20 +133,28 @@ function replaceCdnVersionsInFiles(cdnVersions) {
 function testBuild() {
   announce(`Testing the current build (ENV: ${JSON.stringify(PTOR_ENV, null, 2)})...`);
 
-  const installCmd = `${utils.getExecutable('yarn', true)} install`;
-  const wdrManagerCmd = `${utils.getExecutable('webdriver-manager')} update`;
-  const httpServerCmd = `${utils.getExecutable('http-server')} -p ${PTOR_PORT} ${DST_DIR}`;
-  const protractorCmd = `${utils.getExecutable('protractor')} ${PTOR_CONF}`;
-
   const protractorOptions = {
     env: Object.assign(process.env, PTOR_ENV),
     stdio: 'inherit'
   };
 
-  const installPromise = chain(Promise.resolve(), installCmd);
-  const wdrManagerPromise = chain(installPromise, wdrManagerCmd);
-  const httpServerPromise = chain(wdrManagerPromise, httpServerCmd);
-  const protractorPromise = chain(wdrManagerPromise, protractorCmd, protractorOptions);
+  const installCmd = `${utils.getExecutable('yarn', true)} install`;
+  const httpServerCmd = `${utils.getExecutable('http-server')} -p ${PTOR_PORT} ${DST_DIR}`;
+  const wdrManagerCmd = `${utils.getExecutable('webdriver-manager')} update`;
+  const protractorCmd = `${utils.getExecutable('protractor')} ${PTOR_CONF}`;
+
+  let httpServerPromise;
+  let protractorPromise;
+
+  if (process.env.TRAVIS) {
+    httpServerPromise = chain(Promise.resolve(), httpServerCmd);
+    protractorPromise = chain(Promise.resolve(), protractorCmd, protractorOptions);
+  } else {
+    const installPromise = chain(Promise.resolve(), installCmd);
+    const wdrManagerPromise = chain(installPromise, wdrManagerCmd);
+    httpServerPromise = chain(wdrManagerPromise, httpServerCmd);
+    protractorPromise = chain(wdrManagerPromise, protractorCmd, protractorOptions);
+  }
```

Comment:
Lines 146-157 might be a little easier to follow with something like:

```js
const setupPromise = process.env.TRAVIS ?
  Promise.resolve() :
  chain(chain(Promise.resolve(), installCmd), wdrManagerCmd);
const httpServerPromise = chain(setupPromise, httpServerCmd);
const protractorPromise = chain(setupPromise, protractorCmd, protractorOptions);
```

Reply:
Maybe ... although I don't think the double chain is a bit weird

Reply:
Ha! I knew you would say that :smiley: It definitely is.
How about:

```js
let setupPromise = Promise.resolve();
if (!process.env.TRAVIS) {
  setupPromise = chain(setupPromise, installCmd);
  setupPromise = chain(setupPromise, wdrManagerCmd);
}
const httpServerPromise = chain(setupPromise, httpServerCmd);
const protractorPromise = chain(setupPromise, protractorCmd, protractorOptions);
```

(Basically, I want to able to tell the difference between Travis vs local easily.)

Not important though. Feel free to leave it as is.